src/database.py

```diff
@@ -13,12 +13,12 @@ def __setstate__(self, state):
 
   def AddNewPlayer(self, player):
     if player in self.players:
-      print("Nice try NERD this player already exists!")
+      #print("Nice try NERD this player already exists!")
```

Comment:
uncomment this and below

Reply:
when adding 24 matches it trashes up the output on cmd line

Reply:
every match it adds every player again

Reply:
ahhh true. let's delete it then

Reply:
this still needs to be deleted